base/docs/helpdb/Base.jl

```diff
@@ -914,9 +914,9 @@ generate an array of such random numbers.
 randexp
 
 """
-    chop(string)
+    chop(string, i = 1)
```

Comment:
P.S. with all these non traditional indexing changes, does it still make sense to annotate i as a integer?